frontend/src/components/smart/PvPArenaPreparation.vue

```diff
@@ -165,7 +165,18 @@
             <span>{{ tierTopRankers[2] && tierTopRankers[2].rank || 'N/A'}}</span>
           </li>
         </ul>
-        <!-- <a href="#" class="rankings">View all rankings</a> -->
+        <ul class="topPlayersList">
+          <li class="header">
+            <span>Current Season</span><span>Restarts In</span>
```

Comment:
After it's live someone will come back and clean all the translations! I'm watching...

Reply:
Will be done